.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,34 @@
+          node-version: '20'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run tests with coverage
+        run: npm test -- --coverage --watchAll=false
+
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ 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 }}
```

actionlint (1.7.7)

undefined variable "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". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)